test/vm.install

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh -eux
+
+TEST_SOURCE="$1"
+
+# Grow root partition to make room for images. This only works on Fedora right now.
+echo ", +" | sfdisk -N 2 -f /dev/vda
+partprobe
+pvresize /dev/vda2
+lvresize fedora/root -l+100%FREE -r
+
+tmpdir=$(mktemp -d $PWD/srpm-build.XXXXXX)
+tar -xaf $TEST_SOURCE -O $(basename --suffix .tar.gz $TEST_SOURCE)/lorax.spec > $tmpdir/lorax.spec
+rpmbuild -bs \
+         --quiet \
+         --define "_sourcedir $(dirname $1)" \
+         --define "_specdir $tmpdir" \
+         --define "_builddir $tmpdir" \
+         --define "_srcrpmdir `pwd`" \
+         --define "_rpmdir $tmpdir" \
+         --define "_buildrootdir $tmpdir/.build" \
+         $tmpdir/lorax.spec
+
+srpm=$(rpm --qf '%{Name}-%{Version}-%{Release}.src.rpm\n' -q --specfile $tmpdir/lorax.spec | head -n1)
+rm -rf $tmpdir
+
+mock --installdeps $srpm
+mock --offline --resultdir build-results --rebuild $srpm
```

Comment:
Please run these two as user "builder". This may explain the permission errors you saw.